data_structures/queue/queue_with_stacks.py

```diff
@@ -0,0 +1,13 @@
+class Queue(object):
```

Comment:
This file should be in its own directory, not inside the `queue` directory. You can make a directory for it in the `challenges` directory and make a link to the `stack.py` and `node.py` files so that you can use them.